### PR TITLE
Fix: Missing profiles.yml and whitespace+naming nits

### PR DESCRIPTION
**Pull request by gwenwindflower**
We lost the profiles.yml in the wash splitting the last PRs, this re-adds it and fixes some nits from a comb through of the repo looking for issues.

### CODE REVIEW

File: profiles.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  target: dev
+  outputs:
+    dev:
+      type: snowflake
+      account: "{{ env_var('SNOWFLAKE_SANDBOX_ACCOUNT') }}"
+      user: "{{ env_var('SNOWFLAKE_SANDBOX_USER') }}"
+      authenticator: externalbrowser
+      database: analytics
+      warehouse: transforming
+      schema: dbt_winnie
```
Comment (joellabes) [inline profiles.yml:15]: Should this be a generic name?

**Review by joellabes — DISMISSED** [review]

File: profiles.yml (hunk 1 above)
Comment (gwenwindflower) [reply profiles.yml:15]: great question — interested in your thoughts here:
- i would like for this to be generic
- if it is, as the primary developer on this repo for the time being i have to constantly update it and switch it back when pushing and pulling and that feels a bit wack and error prone
- not including a profiles.yml feels bad though, i like to package one in the repo, especially when in this case it's only so sqlfluff can run the dbt templater.

soooo given those competing considerations any ideas?

maybe leave it specific for add more comments around the stuff you should change? something else i haven't thought of?

Comment (joellabes) [reply profiles.yml:15]: A comment would be fine! Was just checking you hadn’t forgotten to change it as opposed to disliking it.

Comment (gwenwindflower) [reply profiles.yml:15]: okay i addressed this! i added a heavily commented example profile next to the profile, and expanded the README to explain more about SQLFluff and pre-commit and why there has to be a profile in this project at all. Thanks for the push on this!

**Review by joellabes — APPROVED** [review]
lgtm!

**Pull request merged**